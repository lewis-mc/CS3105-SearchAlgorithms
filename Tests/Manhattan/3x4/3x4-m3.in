3 4 1 2 3 4 5 6 8 0 9 10 7 11