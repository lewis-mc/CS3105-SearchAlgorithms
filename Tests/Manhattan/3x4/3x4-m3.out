1 3 8 7 11